### ISSUE TITLE
Error when saving entries when there are multiple sites

### ISSUE THREAD

**Issue by ep-iain** [engram-design/SuperTable#126]
### Description

I'm getting the following error when trying to save a new entry with a Super Table field:

```
Unknown Property – yii\base\UnknownPropertyException
Setting unknown property: verbb\supertable\records\SuperTableBlockRecord::ownerSiteId
```

If I try to edit an entry that has already been created, after adding a Super Table field to the entry type, I get:

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'supertableblocks.ownerSiteId' in 'field list'
```

The Craft install this occurs on is using multiple sites.

### Steps to reproduce

1. Create multiple sites
2. Add a Super Table field to an entry type
3. Save a new entry of that type

### Additional info

- Plugin version: 2.0.3
- Craft version: 3.0.0-RC10

**Comment by ep-iain** [comment]
Okay, turns out this is completely unrelated to multisite. This error is actually occurring because this Craft install was built on an upgraded Craft 2 site that had a previous version of Super Table installed.

There was an old 'supertableblocks' table (with the wrong columns) which meant that the installation migration was not running.

I've fixed it by:
- deleting any super table fields
- uninstalling super table
- manually deleting the tables: 'supertableblocks' and 'supertableblocktypes'
- removing any references to super table blocks in 'craft_fields'
- reinstalling super table

**Comment by engram-design** [comment]
@ep-iain Thanks for the investigation on this, although we'll strive to work out a fix rather than deleting everything and starting from scratch. We'll try and reproduce this locally.

**Comment by engram-design** [comment]
This should be fixed in the [2.0.4 release](https://github.com/verbb/super-table/releases/tag/2.0.4).
